cluster/gce/util.sh

```diff
@@ -745,16 +745,17 @@ function check-cluster() {
       if [[ ${elapsed} -gt ${KUBE_CLUSTER_INITIALIZATION_TIMEOUT} ]]; then
           echo -e "${color_red}Cluster failed to initialize within ${KUBE_CLUSTER_INITIALIZATION_TIMEOUT} seconds.${color_norm}" >&2
           if [[ ${KUBE_TEST_DEBUG-} =~ ^[yY]$ ]]; then
-            local tmp_log="$(mktemp)"
-            local file
-            for file in /var/log/startupscript.log /var/log/kube-apiserver.log; do
-              echo "${MASTER_NAME}:${file} contents:"
-              if gcloud compute copy-files --project "${PROJECT}" \
-                --zone "${ZONE}" "${MASTER_NAME}:${file}" "${tmp_log}"; then
-                cat "${tmp_log}"
-              fi
+            local logdir=/var/log
+            local basename
+            for basename in startupscript kube-apiserver; do
+              local src="${logdir}/${basename}.log"
+              echo "Preserving ${MASTER_NAME}:${src}"
+              gcloud compute copy-files \
+                -project "${PROJECT}" --zone "${ZONE}" \
+                "${MASTER_NAME}:${src}" \
+                "${E2E_REPORT_DIR}/${basename}.${MASTER_NAME}.log.$(date +'%Y%m%d-%H%M%S')" \
```

Comment:
> to be more consistent with what test/e2e/core.go does, maybe swap ${MASTER_NAME} and ${basename}?
> 
> (We also don't save the datetime there, but I could go either way on that.)

I'll switch to putting hostname first and ditch the timestamps, but leave a TODO that maybe we should revisit.

In both cases I was just trying to drag over what seemed useful/applicable from how we name google3 logfiles and trying to avoid perpetuating this:

```console
% ls -ld build-log*
-rw-r----- 1 mml eng   28523 Feb 12 12:07 build-log (10).txt
-rw-r----- 1 mml eng   13924 Feb 12 14:10 build-log (11).txt
-rw-r----- 1 mml eng   58968 Jan 28 15:18 build-log (1).txt
-rw-r----- 1 mml eng 1284545 Jan 28 16:05 build-log (2).txt
-rw-r----- 1 mml eng  858403 Feb  2 17:06 build-log (3).txt
-rw-r----- 1 mml eng   13742 Feb 11 10:35 build-log (4).txt
-rw-r----- 1 mml eng   17319 Feb 11 14:27 build-log (5).txt
-rw-r----- 1 mml eng   13694 Feb 11 14:29 build-log (6).txt
-rw-r----- 1 mml eng 1685289 Feb 11 17:09 build-log (7).txt
-rw-r----- 1 mml eng 1685289 Feb 11 17:10 build-log (8).txt
-rw-r----- 1 mml eng  280851 Feb 12 11:51 build-log (9).txt
-rw-r----- 1 mml eng    7220 Jan 28 15:16 build-log.txt
```